models/public/mobilefacedet-v1-mxnet/mobilefacedet-v1-mxnet.md

```diff
@@ -0,0 +1,100 @@
+# mobilefacedet-v1-mxnet
+
+## Use Case and High-Level Description
+
+  MobileFace Detection V1 is a Light and Fast Face Detector for Edge Devices (LFFD) model based on Yolo V3 architecture and trained with MXNet. For details see the [repository](https://github.com/becauseofAI/MobileFace) and [paper](https://arxiv.org/pdf/1904.10633.pdf).
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 3.5456       |
+| MParams           | 7.6828        |
+| Source framework  | MXNet\*       |
+
+## Input
+
+### Original model
+
+Image, name - `data`, shape - `[1x256x256x3]`, format -`[BxHxWxC]` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Expected color order -  `BGR`.
+
+### Converted model
+
+The converted model has the same parameters as the original model.
+
+## Output
+
+### Original model
+
+1. The array of detection summary info, name - `yolov30_slice_axis1`,  shape - `1,18,8,8`. The anchor values are `118,157,  186,248,  285,379`.
+
+2. The array of detection summary info, name - `yolov30_slice_axis2`,  shape - `1,18,16,16`. The anchor values are `43,54,  60,75,  80,106`.
+
+3. The array of detection summary info, name - `yolov30_slice_axis3`,  shape - `1,18,32,32`. The anchor values are `10,12,  16,20,  23,29`.
+
+For each case format is `B,N,Cx,Cy`, where
+    - `B` - batch size
+    - `N` - number of detection boxes for cell
+    - `Cx`, `Cy` - cell index
```

Comment:
More correct format is `B,3*BB, Cx,Cy`, where `BB` is a bounding box for face.
You can use any other name instead of `BB`.

Reply:
what is 3 means?

Reply:
Model produces three bounding boxes with according anchors for each cell. 
(so, true format is `B,N,BB,CX,CY`, where N=3 (bbox count), BB=6 (x,y,h,w,conf,class) etc.)

Reply:
See as [example](https://github.com/opencv/open_model_zoo/blob/develop/models/public/yolo-v3-tf/yolo-v3-tf.md)

Reply:
please review again